tide_webform.install

```diff
@@ -220,3 +220,34 @@ function tide_webform_update_8007() {
     $block->save();
   }
 }
+
+/**
+ * Updating service version.
+ */
+function tide_webform_update_8008() {
+  if (\Drupal::moduleHandler()->moduleExists('tide_webform')) {
+    $config_factory = \Drupal::configFactory();
+    $config = $config_factory->getEditable('tide_webform.defaults');
+    $content = $config->get('privacy_statement.content');
+    $content_value = str_replace("8", "9", $content);
+    $config->set('privacy_statement.content', $content_value);
+    $config->save();
+  }
+}
+
+/**
+ * Updating service version on existing webform.
+ */
+function tide_webform_update_8009() {
+  if (\Drupal::moduleHandler()->moduleExists('webform')) {
```

Comment:
hey  @edyuenyw  doesn't need to check whether `webform` exists or not.

Reply:
Thank you @vincent-gao. Will remove it.